dvc/fs/dvc.py

```diff
@@ -459,7 +468,7 @@ def _info(  # noqa: C901, PLR0912
                     continue
 
         if not dvc_info and not fs_info:
-            raise FileNotFoundError
+            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
```

Comment:
Trying to improve error message as we now log the message for `dvc ls`.

I had to do that because you'd see an `unexpected error` in case of `FileNotFoundErrror`.